src/ui/styles/variables.scss

```diff
@@ -0,0 +1,9 @@
+$breakpoint-mobile: 1025px;
+$glimmer-orange: #F5835F;
+$header-height: 50px;
+$overlay-window-max-width: 700px;
+$overlay-side-padding-desktop: 50px;
+$overlay-side-padding-mobile: 30px;
+$overlay-top-padding-desktop: 20px;
+$overlay-top-padding-mobile: 20px;
+$overlay-border-radius: 8px;
```

Comment:
Should this be the same as the buttons? `5px`, it would keep the design more consistent IMO.